Telephone/Telephone.c

```diff
@@ -0,0 +1,154 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <string.h>
+#include <stdlib.h>
+
+#define MAX_SIZE_OF_NAME 80
+#define MAX_SIZE_OF_PHONE 20
+#define MAX_NUM_OF_RECORDS 100
+#define BUFSIZE 128
+
+typedef struct Record {
+    char name[MAX_SIZE_OF_NAME + 1];
+    char phone[MAX_SIZE_OF_PHONE + 1];
+} Record;
+
+int readFromFile(Record* records) {
+    FILE* database = fopen("database.txt", "r");
+    if (database == NULL) {
+        printf("Unable to open file");
+        return -1;
+    }
+    char name[MAX_SIZE_OF_NAME] = {0};
+    char phone[MAX_SIZE_OF_PHONE] = {0};
+    int numRecords = 0;
+    for (int line = 0; line < MAX_NUM_OF_RECORDS; line++) {
+        //Read name
```

Comment:
```suggestion
        // Read name
```
И ниже. Это более-менее общее правило, текст комментария отделяется пробелом от символа комментария. Во всех языках так принято обычно.